lib/minimax.rb

```diff
@@ -0,0 +1,49 @@
+class Minimax
+
+  def find_best_move(board, depth=0, best_score={}, current_player, player1, player2)
+    opponent = set_opponent(current_player, player1, player2)
+    available_squares = board.available_squares
+
+    return score_move(board, depth, current_player, opponent) if 
+      board.stop_playing?(player1, player2)
```

Comment:
I find it confusing that the board needs to know about the players to determine whether it should stop playing. Also, is the concept of "playing" something that a board knows about? Looking at the method declaration, it seems you're asking whether the board is full (a tie) or if a player has won, which I think could be better represented as something like `game.game_over?`.

Reply:
Thanks, Ashley. I agree. I do have a method in Game, which handles this. I'll look into incorporating `game.over?`